Specs/Scene/ModelExperimental/ModelExperimentalAnimationCollectionSpec.js

```diff
@@ -0,0 +1,748 @@
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: interpolationTestUrl,
+      },
+      scene
+    ).then(function (model) {
+      const animationCollection = model.activeAnimations;
+      const animation = animationCollection.add({ index: 3 });
+      expect(animationCollection.get(0)).toBe(animation);
+    });
+  });
+
+  it("remove returns false for undefined", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (model) {
+      const animationCollection = model.activeAnimations;
+      animationCollection.add({ index: 0 });
+      expect(animationCollection.remove(undefined)).toBe(false);
+    });
+  });
+
+  it("remove returns false for animation not in collection", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: animatedTriangleUrl,
+      },
+      scene
+    ).then(function (firstModel) {
+      return loadAndZoomToModelExperimental(
+        {
+          gltf: animatedTriangleUrl,
+        },
+        scene
+      ).then(function (secondModel) {
+        const firstCollection = firstModel.activeAnimations;
+        const animation = firstCollection.add({ index: 0 });
+        const secondCollection = secondModel.activeAnimations;
+        expect(secondCollection.remove(animation)).toBe(false);
+      });
+    });
+  });
+
+  it("remove returns true for animation in collection", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: interpolationTestUrl,
+      },
+      scene
+    ).then(function (model) {
+      const animationCollection = model.activeAnimations;
+      const animationToRemove = animationCollection.add({ index: 0 });
+      animationCollection.add({ index: 1 });
+      expect(animationCollection.length).toBe(2);
+
+      expect(animationCollection.remove(animationToRemove)).toBe(true);
+      expect(animationCollection.length).toBe(1);
+
+      // Should not be true again
+      expect(animationCollection.remove(animationToRemove)).toBe(false);
+    });
+  });
+
+  it("removeAll works", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: interpolationTestUrl,
+      },
+      scene
+    ).then(function (model) {
+      const animationCollection = model.activeAnimations;
+      animationCollection.addAll();
+      expect(animationCollection.length).toBe(9);
+      animationCollection.removeAll();
+      expect(animationCollection.length).toBe(0);
+    });
+  });
+
+  it("update returns false when there are no animations", function () {
+    return loadAndZoomToModelExperimental(
+      {
+        gltf: interpolationTestUrl,
+      },
+      scene
+    ).then(function (model) {
+      const animationCollection = model.activeAnimations;
+      expect(animationCollection.length).toBe(0);
+      expect(animationCollection.update()).toBe(false);
+    });
+  });
+
+  const scratchJulianDate = new JulianDate();
```

Comment:
just noticed this scratch variable in this file. Same comment as before, move this up higher where data is initialized.